ocaml/parsing/extensions_parsing.mli

```diff
@@ -113,9 +119,18 @@ module type AST = sig
   val wrap_desc :
     loc:Location.t -> attrs:Parsetree.attributes -> ast_desc -> ast
 
-  (** Embed a language extension term in the AST with the given name
-      and body (the [ast]).  The name will be joined with dots
-      and preceded by [extension.].  Partial inverse of [match_extension]. *)
+  (** Embed a language extension term in the AST with the given name and body
+      (the [ast]).  The name will be joined with dots and preceded by
+      [extension.].  The location will be made into a ghost location, as it only
+      goes on the generated parts of the AST.
+
+      NB: When the result is wrapped up in an [ast] node by [wrap_desc], the
+      result probably needs to be made [ghost] as well.  The same holds for
+      other wrapping (e.g., wrapping a [pattern] and [expression] in a
+      [value_binding] with [Ast_helper.Vb.mk]); any location on a generated
+      OCaml AST node needs to be [ghost]!
```

Comment:
Might be nice to say what goes wrong if you get this incorrect.

Reply:
Added in 9ffea8fbb